src/style/shadows.scss

```diff
@@ -1,3 +1,4 @@
+@import './functions';
```

Comment:
I want to avoid ever importing anything from the `internal` folder in the stylesheets that go out externally. That's why I moved this file.